### ISSUE TITLE
FW 10.0.x/PGL support

### ISSUE THREAD

**Issue by briaguya** [misson20000/twili#104]
When booting on Atmosphere 0.12.0 on FW 10.0.2 we get a fatal error
```
Error Code: 2001-0123 (0xf601)
Program: 010000000006840
Firmware: 10.0.2 (Atmosphere 0.12.0-master-7bc0250c)
```

after asking in the reswitched discord, leoetlino responded:

> twili needs to be updated to use [PGL](https://switchbrew.org/wiki/PGL_services), as far as I can tell

> this function is (probably) where most of the changes need to be done https://github.com/misson20000/twili/blob/d8b9347deace6a8971d07d5c4bf31cc70e2dbb1f/twili/process/ShellTracker.cpp#L42

**Comment by misson20000** [comment]
I started working on this last week. Made a good bit of progress but not quite finished yet. Code is available on this branch: https://github.com/misson20000/twili/tree/wip-10.0.0. Not sure when I will have the time again to work on this.

**Comment by misson20000** [comment]
Fixed! See v1.4.0_pre1.
